NomadicNibbler/foodtruck-fe
# Feature/header

## What does this PR do?

- Sass was installed and all css files were converted to scss
- The header component was built and styled
   - Mobile first design is implemented
   - Responsive styling was added

## How should it be tested?

- Open up the branch in localhost:3000 and test out the header for all screen sizes
- Check for: 
  - awkward break points
  - issues with contrast? accessibility? 
  - It is a starting point for the color palette as well, feel free to comment on suggestions

## What do the future iterations hold?

- functionality to return to the user sign in form may be put into the header at a later time


## Linked issues: 
 - #16
